examples/nextjs-14-authjs-5/package.json

```
@@ -0,0 +1,34 @@
+{
+  "private": true,
+  "description": "An example project for NextAuth.js with Next.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@radix-ui/react-avatar": "^1.0.3",
```

```suggestion
    "@arcjet/next": "file:../../arcjet-next",
    "@radix-ui/react-avatar": "^1.0.3",
```
I think this is missing